testlapack/dbdsqr.go

```diff
@@ -35,6 +35,8 @@ func DbdsqrTest(t *testing.T, impl Dbdsqrer) {
 			{10, 10, 10, 10, 30, 40, 50},
 			{10, 12, 11, 13, 30, 40, 50},
 			{20, 12, 13, 11, 30, 40, 50},
+
+			{130, 130, 130, 500, 900, 900, 500},
```

Comment:
This came up in debugging. This test adds a case where blocking needs to be used in the subsequent calls. I can remove if desired.